### PR TITLE
New Entry: General One Hot Encoding

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword List

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/one-hot-encoding/one-hot-encoding.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/build-deep-learning-models-with-tensorflow'
+---
+
+**One hot encoding** is a method of encoding categorical variables as binary vectors that can be more readily used by [machine learning](https://www.codecademy.com/resources/docs/general/machine-learning) [algorithms](https://www.codecademy.com/resources/docs/general/algorithm). Algorithms work with numbers such as `0` and `1`, not categorical values like "Texas" or "bicycle." However, the data these algorithms are asked to analyze often have values that are encoded categorically. One hot encoding is a process by which those categorical values in the data can be translated into numbers that can be interpreted by those algorithms.
+
+## Encoding Categorical Values
+
+The process of one hot encoding is as follows:
```
Comment (Dusch4593) [inline content/general/concepts/one-hot-encoding/one-hot-encoding.md:20]: Is this "process" supposed to be sequential? If so, should we turn this list into a numbered list?

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just one suggestion from me on this one 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestion.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM for the next review 👍🏻

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski Just a few minor things from me!

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Thank you, LGTM! 👍

**Comment by Dusch4593** [comment]
@SSwiniarski @KTom101 I think we're ready to merge this one! 😄 But I had a question for everyone regarding "vectors"; should we file an issue to (a) create a new General concept entry about vectors, or (b) edit the existing [General: Array](https://www.codecademy.com/resources/docs/general/array) to include info about how vectors are 1-dimensional arrays?

**Comment by SSwiniarski** [comment]
> @SSwiniarski @KTom101 I think we're ready to merge this one! 😄 But I had a question for everyone regarding "vectors"; should we file an issue to (a) create a new General concept entry about vectors, or (b) edit the existing [General: Array](https://www.codecademy.com/resources/docs/general/array) to include info about how vectors are 1-dimensional arrays?

@Dusch4593, my vote is for a new entry.

**Comment by KTom101** [comment]
> > @SSwiniarski @KTom101 I think we're ready to merge this one! 😄 But I had a question for everyone regarding "vectors"; should we file an issue to (a) create a new General concept entry about vectors, or (b) edit the existing [General: Array](https://www.codecademy.com/resources/docs/general/array) to include info about how vectors are 1-dimensional arrays?
> 
> 
> 
> @Dusch4593, my vote is for a new entry.
> 
> 
> 
> 
@Dusch4593 +1 on what @SSwiniarski said

**Pull request merged**